tools/cli/commands-packages.js

```diff
@@ -2120,7 +2120,7 @@ main.registerCommand({
     } else if (! projectContext.projectConstraintsFile.getConstraint(packageName)) {
       // Check that we are using the package. We don't check if the package
       // exists. You should be able to remove non-existent packages.
-      Console.error(packageName  + " is not in this project.");
```

Comment:
@ramirez530 I'm sorry that I didn't get what are you talking about. Could you make it clear?